sdcm/cluster_docker.py

```diff
@@ -250,8 +250,8 @@ def _create_nodes(self, count, enable_auto_bootstrap=False):
         return new_nodes
 
     def _get_nodes(self):
-        containers = ContainerManager.get_containers_by_prefix(self.node_prefix)
-        for node_index, container in sorted((int(c.labels["NodeIndex"]), c) for c in containers):
+        containers = ContainerManager.get_containers_by_prefix(f'{self.node_prefix}*')
```

Comment:
shouldn't this be only the prefix, and the `get_containers_by_prefix` adds the `*` after the prefix??

Reply:
but approved, because it will work

Reply:
yeah the `get_containers_by_prefix` is the actual place need to be fixed.
(but this is the only usage of it across the code, might as well change it to `get_containers_by_name`)

Reply:
`get_containers_by_name` would be another function, that searches for specific and exact names